generators/Options.fs

```diff
@@ -0,0 +1,31 @@
+module Generators.Options
+
+open System
+open System.IO
+open CommandLine
+open Serilog
+open Humanizer
+
+type Options = {
+  [<CommandLine.Option('e', "exercises", Required = false, 
+    HelpText = "Exercises to generate (if not specified, defaults to all exercises).")>] Exercises : seq<string>;
```

Comment:
For the `HelpText` it might be helpful to have example values for each of these values. For example, it's not clear to be how to specify multiple exercises to be generated here (I'm assuming some sort of delimiter but I'd have to try whitespace, semicolon, etc). For the canonical data directory, can it be a relative pass as well as absolute?

Reply:
@robkeim I think perhaps these things can be explained in a README (which I plan to add later), as they require a little bit more explanation than I expect of a help text.

Reply:
Sounds good @ErikSchierboom!